### ISSUE TITLE
Unable to logg in after user creation

### ISSUE THREAD

**Issue by TobiasDax** [IgnisDa/ryot#23]
Would love to test this but i am currently unable to log in.

I setup the docker version with the sample config.json and when i open the webUI i can create a new user.
i tried this several times with with different user names and passwords. Each te i can see the logs and it says
```
2023-05-13T08:35:36.383003Z  INFO ryot: Using database backend: "SQLite"
2023-05-13T08:35:38.092251Z  INFO ryot: Listening on 0.0.0.0:8000
2023-05-13T08:36:07.868785Z  INFO ryot::background: Running jobs after user creation
2023-05-13T08:38:57.535388Z  INFO ryot::background: Running jobs after user creation
2023-05-13T08:40:00.185500Z  INFO ryot::background: Running user and metadata association cleanup
```
(tried to setup two new users here)

when i go to the login screen and try to log in i only get

```
Authorization error
You are not logged in
```

Tried this in Edge and Chrome on windows. the Docker is running on UNRAID

**Comment by IgnisDa** [comment]
Thanks for the report! Can you give the following information:

- Open the dev tools and check if there is a cookie named `auth` present when you login

**Comment by TobiasDax** [comment]
hi, no cookie called `auth` is being created when i try to login.

I checked the errors also and saw that all my Browsers are blocking cookies from insecure sites:

`Insecure sites can't set cookies with the 'secure' directive.`

So i guess i have to create a cert for it first?

**Comment by IgnisDa** [comment]
I released a `v0.0.20` with a relevant setting for this. Can you test again using this docker image? Don't forget to use the updated sample config (which I also updated).

**Comment by TobiasDax** [comment]
ok, i am able to login, thanks for the quick help. :)
